### ISSUE TITLE
CGAN using Keras is different with it using Pytorch. 

### ISSUE THREAD

**Issue by Tenyn** [eriklindernoren/Keras-GAN#203]
In keras the label is the real label, but in pytorch it is the random label.

**Comment by vickcn** [comment]
Thank for your great works!! However, I'm still confused about why in the training process there seems to be no working about shuffling the real images with arbitrary labels? But the algorithm does product exactly images with the correct labels.
